### ISSUE TITLE
Feature Request: Star Wars / Genesys dice

### ISSUE THREAD

**Issue by source-toad** [javalent/dice-roller#142]
The addition of functionality for Star Wars / Genesys dice would be awesome :)

**Comment by sigrunixia** [comment]
Hi @source-toad
I'm replying to this for the upgraded Dice Roller documentation. Would you be able to provide some examples of how you use the Dice Roller integration in a Genesys setup?

**Comment by source-toad** [comment]
How I would use it for Genesys? I don’t think this is implemented yet..

**Comment by sigrunixia** [comment]
Ah! I saw the assets inside the source files so I assumed it was in there. It might _just_ be the dice though.

So a different question then. To you, what symbols, or letters make sense for the different dice representations? For an example, Fudge has XdF and Fantasy Age has 1dS.

**Comment by source-toad** [comment]
Yeah, javalent hasn’t had the chance to finish it off yet, for obvious reasons. We’re looking at ‘dice: 1a1p1c1d1s1b’ syntax, for Ability, Proficiency, Challenge, Difficulty, Setback and Boost dice.

**Comment by sigrunixia** [comment]
I'll assume that for now and get some draft notes started. :)

**Comment by saif-ellafi** [comment]
Would be awesome to have Genesys dice here, as well as FATE dice :)

**Comment by saif-ellafi** [comment]
I know it isn't a lot of help but there is a GitHub here with Genesys dice resources for a digital dice roller

https://github.com/D1SoveR/genesys-dice-roller

and this one has the resources for the 3D dice roller in Foundry for Genesys (and for star wars)

https://github.com/StarWarsFoundryVTT/StarWarsFFG/tree/main/images/dice/genesys

**Comment by valentine195** [comment]
Hey, reminded me to give a little update here. Now that individual dice request their own render in v10, it will be a lot easier to get this implemented.

**Comment by saif-ellafi** [comment]
That would be so awesome :) - the above links I shared are more meaningful for the resources they have, but to actually understand the logic I think the dice pool setup and dice meaning is better exposed here: https://rpgdice.cenix.dk

**Comment by FilBot3** [comment]
There is also this GitHub repository that has HTML approximations of the Genesys symbols.
- https://github.com/themensch/FFG-Genesys-Glyphs-and-Dice/blob/master/Genesys_Unicode_HTML5_approximations.md

**Comment by themensch** [comment]
There are also images in this repo!

**Comment by source-toad** [comment]
Just commenting here that this has been implemented. It might not let me close it, but im going to try.

**Comment by source-toad** [comment]
Feel free to reach out on the ObsidianMD discord if anyone needs support, but there are some instructions in this repo's readme now too.
